src/FormParent.jsx

```diff
@@ -59,22 +60,102 @@ const FormParent = () => {
     });
   };
 
-  // Call asynchronous functions upon changes to state variables.
+  /**
+   * Get exon's data
+   * @param {*} txAc transcript accession
+   * @param {*} startExon starting exon number
+   * @param {*} endExon ending exon number
+   * @param {*} startExonOffset starting exon's offset
+   * @param {*} endExonOffset ending exon's offset
+   * @param {*} gene gene symbol
```

Comment:
These are all strings right?

```suggestion
   * @param {string} txAc transcript accession
   * @param {string} startExon starting exon number
   * @param {string} endExon ending exon number
   * @param {string} startExonOffset starting exon's offset
   * @param {string} endExonOffset ending exon's offset
   * @param {string} gene gene symbol
```

Reply:
Could be string or number. Updated in most recent commit!